enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/constants/RouterAccessLogConstants.java

```diff
@@ -23,4 +23,5 @@
  */
 public class RouterAccessLogConstants {
     public static final String ORIGINAL_PATH_DATA_NAME = "originalPath";
+    public static final String ORIGINAL_HOST_DATA_NAME = "originalHost";
```

Comment:
Minor comment. These can be changed to `downstreamHost` and `downstreamPath` to be more clear

Reply:
It is better not to change `originalPath` I guess, Since it being already used it may break the expected functionality for a while. (Until we push overlay changes)